charts/portworx/v2.8.1/templates/portworx-csi.yaml

```diff
@@ -65,32 +116,70 @@ spec:
       containers:
         - name: csi-external-provisioner
           imagePullPolicy: Always
-          image: "{{ template "px.getcsiProvisioner" . }}/csi-provisioner:v0.2.0"
+          image: {{ template "px.getCSIProvisionerImage" . }}
+          args:
+            - "--v=3"
+            {{- if (semverCompare "< 1.19.0" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
this should be only when the provisioner is `csi-provisioner:v1.6.1-1` aka a major version of v1. This means it should use the same if clause as csi-provisioner:v1.6.1-1, which is k8s version of < 1.17.

The else should declare the following for k8s 1.17 and later -
```
            - "--v=3"
            - "--csi-address=$(ADDRESS)"
            - "--leader-election=true"
            - "--default-fstype=ext4"
```

Reply:
i would just redo this args section to match the PX-installer if/else clauses, easier to maintain when they're identical